sdcm/cluster.py

```diff
@@ -3353,13 +3352,16 @@ def execute_cmd(cql_session, entity_type):
             else:
                 raise ValueError(f"The following value '{entity_type}' not supported")
 
-            session.default_fetch_size = 1000
-            session.default_consistency_level = ConsistencyLevel.ONE
-            execute_result = session.execute_async(cmd)
-            fetcher = PageFetcher(execute_result).request_all()
-            current_rows = fetcher.all_data()
+            cql_session.default_fetch_size = 1000
+            cql_session.default_consistency_level = ConsistencyLevel.ONE
+            execute_result_future = session.execute_async(cmd, timeout=120)
```

Comment:
@aleksbykov what happens if we use it in places where we have more than 1000 results?
Or even 1000 results take too long to come back, and we need smaller pages.

(I don't know if we do it or not).

Isn't the original problem you're trying to solve is much simpler?
We just need to make sure the execute_async completed returned.